app/web/components/CenteredSpinner/CenteredSpinner.tsx

```diff
@@ -0,0 +1,30 @@
+import { CircularProgress, styled } from "@mui/material";
+
+interface CenteredSpinnerProps {
+  minHeight?: string;
+}
+
+interface StyleProps {
+  minHeight?: string;
+}
+
+const StyledCenteredLoaderContainer = styled("div")<StyleProps>(
+  ({ theme, minHeight }) => ({
+    //minimal-effort reduction of layout shifting
```

Comment:
Just out of curiousity what does this comment mean?

Reply:
It was a comment in the FullPageLoader component so I just copied it over since it was a very similar component. It's not very explanatory though so will delete